src/Containers/SignUpForm/SignUpForm.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import './SignUpForm.scss';
+
+interface Props {
+  isLogin: boolean,
+  toggleTab: (login: boolean) => void
+}
+
```

Comment:
Should we move this into the interfaces file?

Reply:
No, this is a specific interface for this component only.